stateMachine.js

```diff
@@ -61,7 +61,9 @@
 
       function buildEventShortcut(name) {
         states[name] = states[name] || function () {
-          states.trigger(name);
+          var args = Array.prototype.slice.call(arguments);
+          var arr = args.sort();
```

Comment:
Why this sort call? This is really unexpected in my experience, I'd expect the order or arguments to never change?